src/components/Header/index.tsx

```diff
@@ -35,9 +35,11 @@ export type IconFormatted = {
 const NavLinks = ({
   mappedLinks,
   webProperty,
+  active,
 }: {
   mappedLinks: MappedLink<LinkFormatted<IconFormatted>>[];
   webProperty: "originprotocol" | "ousd" | "story";
+  active: string;
```

Comment:
where is active being set? is this data we're getting from the CMS?

Reply:
it would just be passed in directly on the website

Reply:
okay gotcha - let's also add a comment on what's meant by active in the props